### PR TITLE
#8 setup sentry.io for error reporting

### PR DESCRIPTION
**Pull request by AmirolAhmad**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/kldx/main-apps/pull/9" target="_blank">ReviewNinja

File: Gemfile [hunk 1]
```diff
@@ -18,6 +18,7 @@ gem 'jquery-turbolinks'
 gem 'react-rails'
 gem 'friendly_id', '~> 5.1.0'
 gem 'bootstrap-sass', '~> 3.3.6'
+gem 'sentry-raven'
```
Comment (AmirolAhmad) [inline Gemfile]: !fixed

Comment (AmirolAhmad) [inline Gemfile]: nice

Comment (AmirolAhmad) [inline Gemfile]: !fix

**Pull request merged**